util/src/main/java/io/kubernetes/client/util/Yaml.java

```diff
@@ -201,7 +201,7 @@ public static void dump(Object object, Writer writer) {
    * @param data The list of YAML API objects
    * @return A String representing the list of YAML API objects.
    */
-  public static String dumpAll(Iterator<? extends Object> data) {
```

Comment:
Now that we have `KubernetesType` I think this should be `? extends KubernetesType`

Reply:
Done